regfilter/regfilter.py

```
@@ -1,15 +1,62 @@
+            except discord.HTTPException as e:
+                if hasattr(e, 'status') and e.status == 429:
+                    retry_after = e.response.headers.get('Retry-After')
+                    if retry_after is None:
+                        logging.warning(f"[Regfilter] HTTP 429 but no Retry-After header: {e}")
+                        continue
+                    wait_time = float(retry_after) + 0.1
+                    logging.warning(f"[Regfilter] Rate limited, waiting {wait_time}s")
+                    await asyncio.sleep(wait_time)
+                    try:
+                        await action(item)
+                    except Exception as e2:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)